Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -391,7 +391,7 @@ class CombinatorialKalmanFilter {
             ACTS_ERROR("Error while acquiring bound state for target surface: "
                        << res.error() << " " << res.error().message());
             result.lastError = res.error();
-          } else {
+          } else if (!result.activeTips.empty()) {
```

Comment:
would it make sense to have this check at least on layer higher? I think there is no need to bind the parameters to a surface, when we don't have active tips to handle.

however, I was wondering, how we could end up with this in the first place. I think, only way is to start with an empty tip in the first place. so maybe check this before even calling the actor?

Reply:
`activeTips` is always empty when you start the finding. if you find no measurement and hit the target surface you will end up in this situation

I don't think we want to inverse the checks because logically we want to have the parameters but the current implementation does not allow to pass them through. This is improved with https://github.com/acts-project/acts/pull/3161